model/import/Repository/TemplateRepositoryInterface.php

```diff
@@ -83,7 +83,7 @@ interface TemplateRepositoryInterface
                 'header' => 'optional',
                 'value' => 'one_of_columns_or_empty:choice_[1-99]',
             ],
-            'metadata_[a-z0-9\-_]+' => [
+            'metadata_[A-Za-z0-9\-_]+' => [
```

Comment:
WDYT about keeping this regex within some central place rather than spread hard coded multiple times within code?

Reply:
Sure, but I would like to cover all regex, not just this one, so maybe we could have as a tech debt. I will have to backport this after, so I am trying to avoid big changes.